Tasks/VsTest/versionFinder.ts

```diff
@@ -0,0 +1,108 @@
+import tl = require('vsts-task-lib/task');
+import path = require('path');
+import Q = require('q');
+import models = require('./models')
+
+var regedit = require('regedit');
+var xml2js = require('xml2js');
+
+export function locateVSTestConsole(testConfig): Q.Promise<models.ExecutabaleInfo> {
```

Comment:
>locateVSTestConsole [](start = 16, length = 19)

registry based locator should be a separate class and willow should be separate class..isn't it possible ?